src/org/ods/orchestration/DeployStage.groovy

```diff
@@ -36,7 +37,8 @@ class DeployStage extends Stage {
             // not work on agent nodes yet.
             if (agentPodCondition) {
                 script.node {
-                    script.sh "cp -r ${standardWorkspace}/docs ${script.env.WORKSPACE}/docs"
+                    script.sh "cp -r ${standardWorkspace}/docs ${script.env.WORKSPACE}"
+                    script.sh "cp -r ${standardWorkspace}/projectData ${script.env.WORKSPACE}"
```

Comment:
Are these path changes intentional?

Reply:
@michaelsauter I asked myself the same question. With `cp -r`, a copy operation from `$pathA/$dirA` to `$pathB/$dirA` is be redundant. Therefore, you can also write it as `cp -R $pathA/$dirA $pathB`. `$dirA` is nevertheless created inside `$pathB` unless the source would have a trailing slash, as in `$pathA/$dirA/`.